backend/src/constants.ts

```diff
@@ -1,7 +1,10 @@
 import os from 'os'
 import path from 'path'
 
+//General
 export const ENVIRONMENT = process.env.NODE_ENV || 'production'
+export const PRODUCT_NAME = 'Desktop'
+export const MANUFACTURER_NAME = 'Remote.it'
```

Comment:
remote.it without capital

Reply:
updated

Reply:
I don't see any changes, sure you pushed?

Reply:
pushed